content/soarian/r4/foundation/documents/document-reference.md

```diff
@@ -0,0 +1,151 @@
+---
+title: DocumentReference | R4 API
+---
+
+# DocumentReference
+
+* TOC
+{:toc}
+
+## Overview
+
+The DocumentReference resource is used to reference a clinical document for a patient in the health system. This resource supports reading clinical notes and returning references to retrieve Continuity of Care Documents (CCD).
+
+Soarian Clinicals<sup>®</sup> supports a read-only Application Programming Interface (API). This API accepts `GET` and `POST` based [search] and `GET` based [read] interactions. The response represents the most current information about the patient that is charted in Soarian Clinicals<sup>®</sup> at the time of the query. 
+
```

Comment:
I think the API currently supports only 'GET' parameters

Reply:
we do support 'GET' and 'POST' both the methods.